### ISSUE TITLE
about the traininig

### ISSUE THREAD

**Issue by hezw2016** [MaybeShewill-CV/attentive-gan-derainnet#15]
Hi,
thx a lot for your excellent code. It helps me a lot.
But, I have some questions about the training.
1. the initial learning rate: you said that "In my experiment, the training epochs are 200010, batch size is 1, the initialized learning rate is 0.001.", However, when I checked the config file, I found that it was 0.002. When I trained with 0.002, the D_loss was very large. I guess the initial learning rate is quite important. So I wonder what is the exact value?
2. When you are training your model, did you crop the original image into small patches? Did you aug the images or not? I think directly resizing the size into 240x360 is not a good choice.
Thx again for your excellent code.

Zewei

**Comment by hezw2016** [comment]
@MaybeShewill-CV 
Thx for your reply.
I will re-train the model to see whether the problem still exists or not. If so, I will provide some details. : )

**Comment by MaybeShewill-CV** [comment]
@hezw2016 If you re-train a model with the cropped patches you're welcome to share the result here. It may also help me and other people:)

**Comment by hezw2016** [comment]
@MaybeShewill-CV  OK, no problem, it may take several days on my 1070 GPU. HaHa

**Comment by hezw2016** [comment]
![d_loss](https://user-images.githubusercontent.com/18389932/48366871-ae51ee80-e674-11e8-9ccf-f0b4acea72e6.png)
@MaybeShewill-CV Hi, the D_loss problem happens again.
At the begining, D_loss is around 15.94, then the D_loss suddenly becomes very large.
here is some details:
++++++++++++++++++++++++
I1112 12:18:20.574566 21056 train_model.py:209] =====>Epoch: 6676 D_loss: 225723.28125 G_loss: 1.69909 Ssim: 0.76703 Cost_time: 1.06179s
I1112 12:18:21.646596 21056 train_model.py:209] =====>Epoch: 6677 D_loss: 120077.87500 G_loss: 1.64597 Ssim: 0.79087 Cost_time: 1.07203s
I1112 12:18:22.716292 21056 train_model.py:209] =====>Epoch: 6678 D_loss: 82806.89844 G_loss: 2.30736 Ssim: 0.72235 Cost_time: 1.06869s
I1112 12:18:23.811926 21056 train_model.py:209] =====>Epoch: 6679 D_loss: 102868.09375 G_loss: 0.90170 Ssim: 0.66528 Cost_time: 1.09463s
I1112 12:18:24.892297 21056 train_model.py:209] =====>Epoch: 6680 D_loss: 118568.27344 G_loss: 2.06072 Ssim: 0.77135 Cost_time: 1.07936s
I1112 12:18:25.973346 21056 train_model.py:209] =====>Epoch: 6681 D_loss: 40991.67969 G_loss: 2.10659 Ssim: 0.71723 Cost_time: 1.08005s
I1112 12:18:27.051629 21056 train_model.py:209] =====>Epoch: 6682 D_loss: 153677.87500 G_loss: 1.27149 Ssim: 0.80044 Cost_time: 1.07728s
I1112 12:18:28.111481 21056 train_model.py:209] =====>Epoch: 6683 D_loss: 119695.32031 G_loss: 2.99901 Ssim: 0.57306 Cost_time: 1.05684s
++++++++++++++++++++++++++++

**Comment by MaybeShewill-CV** [comment]
@hezw2016 ok I'will check it. The tfevents file will be uploaded later:)

**Comment by huoguangdiandian** [comment]
我也遇到同样的问题，D_loss值很大，能达到十几万，请问这个问题怎么解决？是不是跟log函数的参数有0存在的原因，比如log(0)？
